types/iobroker/iobroker-tests.ts

```diff
@@ -677,10 +677,6 @@ enumObj.common.members && enumObj.common.members.map(() => 1);
 adapter.clearTimeout(adapter.setTimeout(() => {}, 10));
 adapter.clearInterval(adapter.setInterval(() => {}, 10));
 // $ExpectError
-clearTimeout(adapter.setTimeout(() => {}, 10));
-// $ExpectError
-clearInterval(adapter.setInterval(() => {}, 10));
```

Comment:
`adapter.setInterval` returns a branded number and so this no longer fails due to the updates.

Reply:
Hmm the point of this was that timeouts created with the adapter methods must be removed with the corresponding adapter methods.

I'd like to keep that behavior.

Reply:
I updated the iobroker types to remove the `number` type.